packages/ResizeDetector/stories/examples/Debounce.js

```diff
@@ -6,7 +6,7 @@ import ResizeDetector from "../../src";
 export function DebounceSlowStory() {
   return (
     <Resizer initWidth={360} initHeight={64} axis="x">
-      <ResizeDetector debounceDelay={300}>
+      <ResizeDetector isFullWidth debounceDelay={30}>
```

Comment:
Oh, right, you changed it here too 🤦 

I actually deliberately made it really slow so the lag was observable.  But 🤷‍♂️